packages/selenium/test/RunPartial/context.html

```diff
@@ -7,5 +7,7 @@
     <h1>Context Test</h1>
     <div class="include">include me <img src="hello.jpg" /></div>
 	<div class="exclude">exclude me <li>I am not in a list</li></div>
+    <div class="include2">include me <img src="hello.jpg" /></div>
```

Comment:
Not blocking but we should be using the [axe-test-fixtures](https://github.com/dequelabs/axe-test-fixtures/blob/v1/fixtures/context-include-exclude.html) package for testing functionality, should we make a tech-debt ticket for it?